### PR TITLE
Use correct toolchain when building documentation or running doctests via xtask

### PR DESCRIPTION
**Pull request by jessebraham**

### CODE REVIEW

File: xtask/src/main.rs [hunk 1]
```diff
@@ -35,13 +36,15 @@ enum Cli {
     /// Bump the version of the specified package(s).
     BumpVersion(BumpVersionArgs),
     /// Format all packages in the workspace with rustfmt
+    #[clap(alias = "format-packages")]
```
Comment (bugadani) [inline xtask/src/main.rs:39]: We also have [`cargo xfmt`](https://github.com/esp-rs/esp-hal/blob/c19c7fcf01bac8eb82be38a8ddaf603f191a61a5/.cargo/config.toml#L3) if you want to save a few keystrokes :D

Comment (jessebraham) [reply xtask/src/main.rs:39]: It's more just habit than anything at this point 😁 Thanks for pointing that out, though.

**Review by bjoernQ — APPROVED** [review]
LGTM

**Review by bugadani — APPROVED** [review]
Thanks, this will save my sanity I'm pretty sure

**Pull request merged**